kyo-data/shared/src/main/scala/kyo/package.scala

```diff
@@ -0,0 +1,8 @@
+package kyo
+
+private[kyo] inline def isNull[A](v: A): Boolean =
+    v.asInstanceOf[AnyRef] eq null
+
+private[kyo] inline def discard[A](v: A): Unit =
```

Comment:
Should this be by-name? I know the `inline` should take care of that, but it's still a bit odd to see.

Reply:
good catch!